### ISSUE TITLE
Cannot compile app from bitcode

### ISSUE THREAD

**Issue by Gray-Wind** [ITECOMMPAY/paymentpage-sdk-ios#2]
I'm using 1.9.0 `Paymentpage-sdk-ios/Universal`

App compilation returns with this error:

```
ipatool failed with an exception: #<CmdSpec::NonZeroExitException: $ /Applications/Xcode.app/Contents/Developer/usr/bin/python3 /Applications/Xcode.app/Contents/Developer/usr/bin/bitcode-build-tool -v -t /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin -L /var/folders/2z/wsk_6jc952z7ymfrpx11fqgwbzl698/T/ipatool20210423-8327-1y5t30s/thinned-out/arm64/Payload/APP_NAME.app/Frameworks --sdk /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS14.4.sdk -o /var/folders/2z/wsk_6jc952z7ymfrpx11fqgwbzl698/T/ipatool20210423-8327-1y5t30s/thinned-out/arm64/Payload/APP_NAME.app/Frameworks/ecommpaySDK.framework/ecommpaySDK --generate-dsym /var/folders/2z/wsk_6jc952z7ymfrpx11fqgwbzl698/T/ipatool20210423-8327-1y5t30s/thinned-out/arm64/Payload/APP_NAME.app/Frameworks/ecommpaySDK.framework/ecommpaySDK.dSYM --strip-swift-symbols /var/folders/2z/wsk_6jc952z7ymfrpx11fqgwbzl698/T/ipatool20210423-8327-1y5t30s/thinned-in/arm64/Payload/APP_NAME.app/Frameworks/ecommpaySDK.framework/ecommpaySDK
Status: pid 11975 exit 1
```

```
        -= Output =-
        ld: warning: -sdk_version and -platform_version are not compatible, ignoring -sdk_version
        Undefined symbols for architecture arm64:
          "___llvm_profile_runtime", referenced from:
              __hidden#99_ in 003.o
              __hidden#99_ in 004.o
              __hidden#99_ in 005.o
              __hidden#99_ in 006.o
              __hidden#99_ in 007.o
              __hidden#99_ in 008.o
              __hidden#99_ in 009.o
              ...
        ld: symbol(s) not found for architecture arm64
        Exited with 1
        
    Failed to compile bundle: /var/folders/2z/wsk_6jc952z7ymfrpx11fqgwbzl698/T/ecommpaySDK_7ulbh82/ecommpaySDK.arm64.xar

Stderr:

	from /Applications/Xcode.app/Contents/Developer/usr/bin/ipatool:2808:in `block in CompileOrStripBitcodeInBundle'
	from /Applications/Xcode.app/Contents/Developer/usr/bin/ipatool:2747:in `each'
	from /Applications/Xcode.app/Contents/Developer/usr/bin/ipatool:2747:in `CompileOrStripBitcodeInBundle'
	from /Applications/Xcode.app/Contents/Developer/usr/bin/ipatool:3016:in `block in ProcessIPA'
	from /Applications/Xcode.app/Contents/Developer/usr/bin/ipatool:2978:in `each'
	from /Applications/Xcode.app/Contents/Developer/usr/bin/ipatool:2978:in `ProcessIPA'
	from /Applications/Xcode.app/Contents/Developer/usr/bin/ipatool:3928:in `<main>'
```

**Comment by Doomsday46** [comment]
Hi, you need to use the iPhoneOS framework to publish your application.
To connect this framework, you need to specify the following line in the Podfile ->`pod 'Paymentpage-sdk-ios', '1.9.0'` or 
`pod 'Paymentpage-sdk-ios / Iphoneos', '1.9.0'`
The framework is connected with support for iOS and iPadOS devices by default.
For more information about frameworks visit https://developers.ecommpay.com/en/en_sdk_ios.html#en_sdk_ios_import__section_qdl_rqt_wmb.

**Comment by Gray-Wind** [comment]
I see, using this special variant for distribution will rise a need to have different frameworks for debug and production environments, which could lead to some errors. It would be really great if you'll have only universal framework without any alternatives.

**Comment by Doomsday46** [comment]
Hello, this issue has been resolved in version 1.9.2. The framework distribution in .xcframework format. Thank you for your feedback.

**Comment by Gray-Wind** [comment]
I'd like to check it, but the framework is not available for download with neither Swift package nor Cocoapods.

**Comment by Doomsday46** [comment]
We have tested the integration of version 1.9.2 through cocoapods.
![photo_2021-07-12_19-27-18](https://user-images.githubusercontent.com/17257861/125323219-38cbde00-e347-11eb-9c18-79912769f0c9.jpg)
![photo_2021-07-12_19-27-09](https://user-images.githubusercontent.com/17257861/125323257-41bcaf80-e347-11eb-8b18-a60bcc0fe179.jpg)

Can you share more information about your problem? What is the minimum supported version of your application, which version of Xcode are you using?

**Comment by Gray-Wind** [comment]
We still support iOS 11

<img width="777" alt="image" src="https://user-images.githubusercontent.com/4902794/125413886-cd5eec4a-a22f-4ca7-aee0-e01bce7f6750.png">

**Comment by Doomsday46** [comment]
Hello, this issue has been resolved in version 1.9.3. Thank you for your feedback.

**Comment by AlexeyKhrameev** [comment]
if problem reproduce
Please open issue again
